src/providers/local/exit-pool.provider.ts

```diff
@@ -506,6 +511,7 @@ export const exitPoolProvider = (pool: Ref<Pool>) => {
     exitTokenInfo,
     queryExitQuery,
     approvalActions,
+    transactionDeadline,
```

Comment:
Don't think it needs to be exported here?

Reply:
Yep, deleted.